docs/getting_started/tutorial09.rst

```diff
@@ -0,0 +1,59 @@
+Your First CodeRed Website: Part 9 -- Images
+============================================
+
+The Media you add to your site in the CMS is an important part of your page design. Images catch people's eyes,
+and good images make good impressions. Let's go over some ways that you can optimize your images for 
+your CodeRed CMS website. 
+
+Preparing your images for upload
+--------------------------------
+
+The CMS will accept the following file types for images:
+
+* **GIF**
+
+* **JPEG/JPG**
+
+* **PNG**
+
+* **WEBP**
+
+If your image is not saved in any of these formats, you need to convert your image or save your image again
+in one of these formats. 
+
+The CMS also has a file size limitation. If your image file is exceeds **10.0 MB**, it will not be able to be uploaded.
```

Comment:
I would add an important note here... Most people have no concept of file size and will upload large images (e.g. multiple megabytes) because that is the raw stock photo size or something like that. The photo should be resized and realistically never larger than 1MB, ideal size is no more than 200-300 Kilobytes. This is a good balance between resolution and file size. The larger the file size, the slower it your website will load when people visit it. The resolution also impacts this file size... for example a file that is 10,000px by 10,000px is too large... no laptop or mobile screen is actually that high of resolution. Photos should generally be no more than 2,000px (width or height).

Some topics for thought...

Reply:
added